### ISSUE TITLE
Ability top copy styles from elements ID to Classname

### ISSUE THREAD

**Issue by Infusednl** [zionbuilder/zionbuilder#132]
## Summary

Often times when I build a layout I edit the element's ID first since I as I am still trying things out.
In such cases it would save a lot of time if we could copy the settings from an elements ID over to a class or copy styles between different classnames on the same element.

Oxygen Builder has this feature since version 3.5. You can their video on it here:

- https://oxygenbuilder.com/2020/08/25/oxygen-3-5-now-available/

### Motivation

Such a feature is a real time-saver and quickly allows on to move styles to a class afterwards.
So in short:

• Copy styles from ID to classname
• Or copy styles from a classname to another
